atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/LockWatchEteTest.java

```diff
@@ -114,6 +114,7 @@ public void multipleTransactionVersionsReturnsSnapshotAndOnlyRelevantRecentEvent
     }
 
     @Test
+    @ShouldRetry
```

Comment:
was flaking

Reply:
If this test is retried, we could miss legit nondeterministic bugs -- what was the flake and can we fix it?

Reply:
It's a genuine flake. I'm going to probably refactor these ETEs anyway soon enough, so I'm going to keep this rule here.